deep4cast/forecasters.py

```diff
@@ -188,18 +193,30 @@ def predict(self, data, n_samples=100, quantiles=(5, 95)):
             samples.append(self._unstandardize(prediction))
 
         samples = np.array(samples)
-
-        # Turn samples into quantiles for easier display later.
-        lower_quantile = np.nanpercentile(samples, quantiles[0], axis=0)
-        upper_quantile = np.nanpercentile(samples, quantiles[1], axis=0)
-        median_prediction = np.nanpercentile(samples, 50, axis=0)
+        
+        # Calculate mean prediction based on samples.
         mean_prediction = np.mean(samples, axis=0)
-
-        return {'mean': mean_prediction,
-                'median': median_prediction,
-                'lower_quantile': lower_quantile,
-                'upper_quantile': upper_quantile,
-                'samples': samples}
+        
+        if not self.uncertainty:
+            # Return only mean prediction based on one sample when Uncertainty
+            # is False
+            print("Uncertainty is set to False. Only mean prediction is"
```

Comment:
Please don't use a print statement for alerting the user here. Instead set the other statics to 'None' or 'NaN' when there is only one sample, so that code that depends on the presence of these fields downstream can simply handle this via testing for Nones or NaNs.

I also suggest shortening this part of the code by using inline ifs. Example:
``lower_quantile = np.nanpercentile(samples, quantiles[0], axis=0) if self.uncertainty else None``